build.gradle

```diff
@@ -657,6 +657,47 @@ project(':iceberg-delta-lake') {
   }
 }
 
+project(':iceberg-bigquery') {
+  test {
+    useJUnitPlatform()
+  }
+
+  dependencies {
+    implementation project(path: ':iceberg-bundled-guava', configuration: 'shadow')
+    implementation project(':iceberg-gcp')
```

Comment:
does the catalog actually directly depend on this?

Reply:
@talatuyarer I've reopened this as it's not clear to me why this dependency here is needed. I understand that BigQuery would always use GCSFileIO but I don't think this dependency here is required (as nothing in the BigQuery module references anything from the gcp module). We do have the gcp-bundle for users that actually use `GCSFileIO`

Reply:
Sorry i accidentally closed this. I used this dependency while testing against BigQuery Prod. I removed